src/integrationTest/java/uk/gov/hmcts/darts/common/service/ViqHeaderServiceTest.java

```diff
@@ -0,0 +1,76 @@
+package uk.gov.hmcts.darts.common.service;
+
+import lombok.extern.slf4j.Slf4j;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.io.TempDir;
+import org.springframework.beans.factory.annotation.Autowired;
+import uk.gov.hmcts.darts.audio.model.ViqMetaData;
+import uk.gov.hmcts.darts.audio.service.impl.ViqHeaderServiceImpl;
+import uk.gov.hmcts.darts.testutils.IntegrationBase;
+
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static uk.gov.hmcts.darts.audio.service.impl.ViqHeaderServiceImpl.COURTHOUSE_README_LABEL;
+import static uk.gov.hmcts.darts.audio.service.impl.ViqHeaderServiceImpl.END_TIME_README_LABEL;
+import static uk.gov.hmcts.darts.audio.service.impl.ViqHeaderServiceImpl.RAISED_BY_README_LABEL;
+import static uk.gov.hmcts.darts.audio.service.impl.ViqHeaderServiceImpl.REQUEST_TYPE_README_LABEL;
+import static uk.gov.hmcts.darts.audio.service.impl.ViqHeaderServiceImpl.START_TIME_README_LABEL;
+
+
+@SuppressWarnings({"PMD.AvoidInstantiatingObjectsInLoops", "PMD.AssignmentInOperand"})
+@Slf4j
+class ViqHeaderServiceTest extends IntegrationBase {
+
+
+    @Autowired
+    ViqHeaderServiceImpl viqHeaderService;
+
+    @TempDir
+    File tempDirectory;
+
+
+    @Test
```

Comment:
Wondering why we have added this as an `integrationTest` if there are no other spring boot test dependencies?  Do we need both `test` or can we just move this stuff under ViqHeaderServiceImplTest?

Reply:
I wasn't certain what way to go tbh - they looked similar to me, but integrationTest had more verify stuff
Just thought we needed one - so will leave you to decide which way to go makes more sense
It just looked questionable for me